### DISCUSSION TITLE
Add a "trigger delay" per TG, for delaying the start of certain sounds/parts

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#720] [Ideas]
The main idea is to create a travel between TG, and use TG as complete section of a sound.

For exemple TG1 is the intro, then TG2 is the body of the sound, and TG3 is the release.

More complexe sounds could be created by this kind of travel.

**Comment by probonopd** [comment]
What do you mean by "travel"? Can you give an example of what should happen?
Yes, it is quite common to use more than one tone generator to make one more complex sound. 
I am not aware that some functionality in MiniDexed is missing for this. Can you explain please?

**Reply by Onosendaifr** [reply]
When you trigger a performance, all TGs are playing at once (by default). Instead of triggering all TG at once, it could be interesting to delayed a TG by secondes to create a time soundscape, one sound morphing to another one, only by delayed the triggering of a TG, 

Choosing a very slow ADSR could be one solution, but then you loose the transient of a rapid ADSR.

**Reply by probonopd** [reply]
I'd say, let's do it how they did back in the day with the TX816, so let's not introduce incompatible additions to the TGs.
